core/src/main/java/tc/oc/pgm/api/party/Party.java

```diff
@@ -132,4 +137,19 @@ default boolean isParticipating() {
   default boolean isObserving() {
     return !this.isParticipating();
   }
+
+  @Override
+  default Optional<? extends Filterable<? super PartyQuery>> filterableParent() {
+    return Optional.of(getMatch());
+  }
+
+  @Override
+  default Stream<? extends Filterable<? extends PlayerQuery>> filterableChildren() {
```

Comment:
`Iterable` instead of `Stream`

Reply:
`Iterable` is a pain to work with, just return `Collection` and let the callers do .stream on the collection